ocaml/xapi/extauth_plugin_ADwinbind.ml

```diff
@@ -729,10 +729,15 @@ let query_domain_workgroup ~domain =
     workgroup_from_server kdc |> Result.get_ok
   with _ -> raise (Auth_service_error (E_LOOKUP, err_msg))
 
-let config_winbind_damon ?(workgroup = "") ?(netbios_name = "") domain =
+let config_winbind_daemon ?(workgroup = "") ?(netbios_name = "") domain =
   let open Xapi_stdext_unix in
   let smb_config = "/etc/samba/smb.conf" in
   let allow_fallback =
+    (*`allow kerberos auth fallback` depends on our internal samba patch,
+     * this patch disalbe fallback to ntlm by default and can be enabled
+     * Looks like upstream is doing something similar on master with
+     * configration `weak_crypto`, check and replace the internal patch when
+     * upgrade to samba packages with this capacity *)
```

Comment:
There are quite a few typos in the comment here, but the code looks OK now. 'disalbe', 'configration', please fix those prior to merging.

Reply:
There are also a few typos in the commit messages